tests/unittest/_torch/auto_deploy/unit/singlegpu/models/test_hybrid_patches.py

```diff
@@ -1,27 +1,33 @@
 import torch  # noqa
 import torch.export as te
 from torch.export import Dim  # noqa
+import pytest
 
 from tensorrt_llm._torch.auto_deploy.export import apply_export_patches, torch_export_to_gm
 from tensorrt_llm._torch.auto_deploy.llm_args import AutoDeployConfig
 from tensorrt_llm._torch.auto_deploy.transformations._graph import move_to_device  # noqa
 from _model_test_utils import get_small_model_config
 
-MODEL_DIR = "ibm-ai-platform/Bamba-9B-v2"
 # NOTE: find example inputs with the same tokenization length to avoid seq concat.
 EXAMPLE_INPUT = "Mamba is a snake with the following properties:"
-# EXAMPLE_INPUT = "Boa is a snake with the following properties:"
 EXAMPLE_INPUT2 = "Tiger is a cat with the following properties:"
 
 
-def test_bamba_patches():
-    model_on_meta_during_export = True
-    export_func: str = "torch_export_to_gm"
-    use_cache: bool = True
-
+@pytest.mark.parametrize(
+    "model_dir,run_verify_generation",
+    [
+        pytest.param("ibm-ai-platform/Bamba-9B-v2", True),
+        pytest.param("nvidia/NVIDIA-Nemotron-Nano-12B-v2", False),
```

Comment:
Nemotron model definition is buggy and cannot run the stock HF pipeline to generate tokens